assets/src/app.tsx

```diff
@@ -25,6 +25,10 @@ if (window.sentry) {
   Sentry.init({
     dsn: window.sentry.dsn,
   })
+
+  if (window.username) {
+    Sentry.setUser({ username: window.username })
```

Comment:
For what it's worth, it looks like this is the username of the form `mbta-active-directory_emaldonado`, do we think that's good to send to Sentry? It's easy enough to parse out as a human reading it but I'm not sure whether or not we might want to join this later on to some data that has the `emaldonado` or `emaldonado@mbta.com` username format.

Reply:
I thought about that some, and decided that for now, the best course of action was to leave the usernames as raw as possible--we can always translate and/or parse them later if we need to.

Reply:
Makes sense to me!